extensions/amp-story/1.0/interactive-confetti.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {htmlFor} from '../../../src/static-template';
+import {Services} from '../../../src/services';
+import {setStyles} from '../../../src/style';
+
+/**
+ * Generates the template for the confetti wrapper.
+ *
+ * @param {!Element} element
+ * @return {!Element}
+ */
+const buildConfettiWrapperTemplate = (element) => {
+  const html = htmlFor(element);
+  return html`
+    <div
+      class="i-amphtml-story-interactive-confetti-wrapper"
+      aria-hidden="true"
+    ></div>
+  `;
+};
+
+/**
+ * Generates a template for a confetti.
+ *
+ * @param {!Element} element
+ * @return {!Element}
+ */
+const buildconfettiTemplate = (element) => {
+  const html = htmlFor(element);
+  return html` <div class="i-amphtml-story-interactive-confetti"></div> `;
+};
+
+/**
+ * Creates a radial burst of emoji.
+ * The burst begins from the center of the parent element and
+ * animates to the edges of the parent element.
+ * Nodes are removed from the dom at the end of the animation.
+ *
+ * @param {!Element} rootEl
+ * @param {!Window} win
+ * @param {string} confettiEmoji
+ * @return {void}
+ */
+export function emojiConfetti(rootEl, win, confettiEmoji) {
+  const CONFETTI_COUNT = 5;
```

Comment:
Any reason why these are declared as file level constants instead of function-level? I think we should either move them to be file level or use camel case if we want to keep them inside this function.

Reply:
Thanks for pointing this out!
In the future some of these constants might be shared between effect functions (if we create more). 
Shared constants could be made file-level at that time.

I think for now it makes sense to keep them in function since they're not shared elsewhere in the file.